### PR TITLE
Initial version client scopes

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: src/client-scopes/add/NewClientScopeForm.tsx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+            id="kc-description"
+            name="description"
+          />
+        </FormGroup>
+        <FormGroup
+          label={
+            <>
+              {t("protocol")} <HelpItem item="storybook" />
+            </>
+          }
```
Comment (ssilvert) [inline src/client-scopes/add/NewClientScopeForm.tsx:93]: Protocol is a required field.  Default should be openid-connect.

File: src/client-scopes/add/NewClientScopeForm.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+          fieldId="kc-gui-order"
+        >
+          <TextInput
+            ref={register}
+            type="text"
+            id="kc-gui-order"
+            name="attributes.gui_order"
+          />
```
Comment (ssilvert) [inline src/client-scopes/add/NewClientScopeForm.tsx:164]: Add validation.  Must be an integer.

File: src/client-scopes/add/NewClientScopeForm.tsx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+                variant={SelectVariant.single}
+                aria-label="Select Encryption type"
+                isOpen={open}
+              ></Select>
+            )}
+          />
+        </FormGroup>
+        <FormGroup
+          label={t("displayOnConsentScreen")}
+          fieldId="kc-display.on.consent.screen"
```
Comment (ssilvert) [inline src/client-scopes/add/NewClientScopeForm.tsx:119]: Help icon missing.

**Review by ssilvert — CHANGES_REQUESTED** [review]
These screens have the header and breadcrumbs missing.  Are the components for that ready to integrate?

**Comment by ssilvert** [comment]
> These screens have the header and breadcrumbs missing. Are the components for that ready to integrate?

Nevermind.  Didn't see your other PR.  I guess we are just missing breadcrumbs.

File: src/client-scopes/add/NewClientScopeForm.tsx (hunk 2 above)
Comment (edewit) [reply src/client-scopes/add/NewClientScopeForm.tsx]: Guess that is a bug in the old console, there you can save a text as well... changed the type to `number` way nicer then validation.

File: src/client-scopes/add/NewClientScopeForm.tsx (hunk 1 above)
Comment (edewit) [reply src/client-scopes/add/NewClientScopeForm.tsx]: yes fetching the protocols is done with this `server-info` call, guess we need your PR that initializes the console

Comment (ssilvert) [reply src/client-scopes/add/NewClientScopeForm.tsx]: Yes, you are right.

Actually, I'm working whoami right now.  But I guess serverinfo needs to be next on the list.

**Review by ssilvert — APPROVED** [review]

**Comment by srambach** [comment]
@edewit and @ssilvert FYI, I'm looking into the tooltips - as it was yesterday anyway, it doesn't quite match PF recommendations and isn't accessible. I made myself an issue. https://github.com/keycloak/keycloak-admin-ui/issues/120

**Comment by ssilvert** [comment]
@srambach Sounds good.

**Pull request merged**